### DISCUSSION TITLE
USB driver for Win 10 on MacBook Pro M1

### DISCUSSION THREAD

**Discussion by ValonEjupi1** [utmapp/UTM#3627] [UTM Help (macOS)]
How can I install USB driver for Win 10 on MacBook Pro M1?
I have installet the UTM but there I cant read any of USBs.

**Comment by conath** [comment]
We need more information to help you. What is the file name of the Windows 10 installer you used? Please also add a screenshot showing the Windows Device Manager (to open it, press cmd-x while the VM is running and click Device Manager in the menu popup).

**Comment by ValonEjupi1** [comment]
![image](https://user-images.githubusercontent.com/66940674/153836360-a5e74bdd-5196-4e63-b784-7810ec804377.png)
![image](https://user-images.githubusercontent.com/66940674/153836417-a1d91491-c9e7-41b6-91a0-91bb7f8e077a.png)

**Reply by conath** [reply]
All drivers should be included with the [UTM SPICE Tools](https://mac.getutm.app/support/), download this on your Mac and attach it to your Windows VM using the CD button.

**Comment by ValonEjupi1** [comment]
Yes I alredy did this but the driver for USB didnt install succsesfully and I also have problem with the display driver.

**Reply by conath** [reply]
Ok. As a next step I would recommend you download the latest version of Windows. Your version is more than 6 months old and there have been multiple problems reported for some previous Insider versions.

**Comment by ValonEjupi1** [comment]
Where I can finde the last version.Or which version did you mean?

**Reply by conath** [reply]
You can get the latest Windows ARM from UUP Dump websites. I usually use [this one](https://uup.rg-adguard.net). For Windows x86_64, just go to the [Microsoft Website](https://www.microsoft.com/en-us/software-download/windows10ISO).

**Comment by antoxaMC** [comment]
Hi all! i have a same problem. Please help me
<img width="392" alt="Снимок экрана 2022-02-22 в 19 41 02" src="https://user-images.githubusercontent.com/100224425/155177926-cc3e4ad1-44cc-4871-8dcf-d1009f19b6cc.png">
<img width="392" alt="Снимок экрана 2022-02-22 в 19 40 51" src="https://user-images.githubusercontent.com/100224425/155177932-0c419c47-e3b6-42c4-a4d6-d444036f345e.png">
<img width="515" alt="Снимок экрана 2022-02-22 в 19 39 38" src="https://user-images.githubusercontent.com/100224425/155177935-7ba40103-4508-4433-9a1c-79b5313f3b31.png">

**Comment by marekmasek** [comment]
I have the same issue on Win11
<img width="451" alt="image" src="https://user-images.githubusercontent.com/48589322/174813739-7a0ace86-d9ca-4366-94fd-22b89e78bebc.png">

<img width="339" alt="image" src="https://user-images.githubusercontent.com/48589322/174813650-0fd9e8ac-4b74-4741-ba63-73f980837a5d.png">

**Reply by CarloAMG** [reply]
Same here, my goal is to use the SD card reader.

**Comment by bsshazar** [comment]
same here

**Comment by belthesar** [comment]
I also seem to be having similar issues. I believe this is because the SPICE Guest Tools ISO doesn't include aarch64 drivers, and I've installed aarch64 Windows. I'm not sure if the SPICE team has aarch64 drivers (looking around, I didn't find any?), or if the UTM team builds their own, but I've got the wrong ISO loaded?

Edit: On further inspection, it seems that there's only a handful of drivers in the SPICE ISO that don't have aarch64 builds. I'm looking to see if there's a correlation to what's missing to what's reporting needing drivers to what's missing, but I imagine this work has already been done by someone on the project already.

**Comment by junaidanjumuom** [comment]
same problem, any luck finding the solution?
